tests/testproject/TestProject.IntegrationServiceA/MySql/MySqlExtensions.cs

```diff
@@ -14,13 +15,21 @@ private static async Task<IResult> VerifyMySqlAsync(MySqlConnection connection)
     {
         try
         {
-            await connection.OpenAsync();
+            var policy = Policy
+                .Handle<MySqlException>()
+                // retry every second for 60 seconds
+                .WaitAndRetryAsync(60, retryAttempt => TimeSpan.FromSeconds(1));
 
-            var command = connection.CreateCommand();
-            command.CommandText = $"SELECT 1";
-            var results = await command.ExecuteReaderAsync();
+            return await policy.ExecuteAsync(async () =>
+            {
+                await connection.OpenAsync();
```

Comment:
This may not be a blocker given this is a test case. But if `OpenAsync(...)` succeeds but the query fails (for whatever transient reason) then it might just continue to fail because it'll start failing on subseuqnet `OpenAsync(...)` calls.

Reply:
I changed this to only retry on connection open.